pkg/rb2wf/converter.go

```diff
@@ -0,0 +1,586 @@
+package rb2wf
+
+import (
+	"context"
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/airplanedev/cli/pkg/api"
+	"github.com/airplanedev/cli/pkg/logger"
+	libapi "github.com/airplanedev/lib/pkg/api"
+	"github.com/pkg/errors"
+)
+
+const (
+	airplaneSDKVersion = "0.2.21"
```

Comment:
So we don't need to bump this until we release breaking changes:

```suggestion
	airplaneSDKVersion = "^0.2"
```

Reply:
Fixed in https://github.com/airplanedev/cli/pull/823.